MAUI/ImageEditor/customview.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title: Custom view Annotation in .NET MAUI Image Editor control | Syncfusion
+description: Learn about the custom view annotation feature of Syncfusion .NET MAUI Image Editor(SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+# Custom view Annotation in .NET MAUI Image Editor (SfImageEditor)
+
+The ImageEditor control in .NET MAUI allows you to add custom view using the `AddCustomAnnotationView` method with customizable settings.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
+                                   Source="image.jpeg" />
+        <Button Grid.Row="1"
+                Clicked="OnAddCustomAnnotationViewClicked" />
+    </Grid>  
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    private void OnAddCustomAnnotationViewClicked(object sender, EventArgs e)
+    {
+        Image customImage = new Image() { HeightRequest = 100, WidthRequest = 100, Aspect= Aspect.Fill};
+        customImage.Source = ImageSource.FromFile("emoji.png");
+        imageEditor.AddCustomAnnotationView(customImage);
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Customize the appearance
+
+Customize the appearance of Custom view annotation using the [`ImageEditorAnnotationSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html).
+
+* [`AllowDrag`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_AllowDrag): Enables or disables the dragging of custom view annotation. The default value is `true`.
+* [`AllowResize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_AllowResize): Enables or disables the resizing of custom view annotation. The default value is `true`.
+* [`Bounds`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Bounds): Specifies the bounds of the custom view. Position the custom view wherever you want on the image. The value of the text bounds should fall between 0 and 1.
```

Comment:
The value of the view bounds should fall between 0 and 1.

Reply:
updated